crypto/evp/e_des.c

```diff
@@ -149,7 +149,7 @@ static int des_cfb1_cipher(EVP_CIPHER_CTX *ctx, unsigned char *out,
                            const unsigned char *in, size_t inl)
 {
     size_t n, chunk = EVP_MAXCHUNK / 8;
-    unsigned char c[1], d[1];
+    unsigned char c[1], d[1] = { 0 };
```

Comment:
Same comment as https://github.com/openssl/openssl/pull/17896/files#r835007857.

Reply:
Split into two lines for aesthetic appeal.